src/string.cr

```diff
@@ -3500,13 +3500,14 @@ class String
   # Converts underscores to camelcase boundaries.
   #
   # ```
-  # "eiffel_tower".camelcase                                   # => "EiffelTower"
-  # "isolated_integer".camelcase(Unicode::CaseOptions::Turkic) # => "İsolatedİnteger"
+  # "eiffel_tower".camelcase                                            # => "EiffelTower"
+  # "empire_state_building".camelcase(downcase: true)                   # => "empireStateBuilding"
+  # "isolated_integer".camelcase(options: Unicode::CaseOptions::Turkic) # => "İsolatedİnteger"
   # ```
-  def camelcase(options : Unicode::CaseOptions = Unicode::CaseOptions::None)
+  def camelcase(downcase = false, options : Unicode::CaseOptions = Unicode::CaseOptions::None)
```

Comment:
Maybe this argument should be at the end to avoid a breaking change?

Reply:
I moved it to the beginning because I feel like `options` parameters are usually at the end of a signature.

Reply:
`options` is self-descriptive because uses Enum. `downcase` should either use Enum, or be a named argument (appear after `*,`), otherwise people will write `str.camelcase(true)`.

Reply:
@konovod can you take another look at this?

Reply:
Looks 👍 now.